crates/gooey-raster/src/lib.rs

```diff
@@ -106,6 +108,16 @@ where
     fn cursor_moved(&mut self, event: MouseEvent, context: &mut dyn AnyRasterContext) {
         T::cursor_moved(self, event, context);
     }
+
+    fn key_down(&mut self, event: KeyEvent, context: &mut dyn AnyRasterContext) {
+        T::key_down(self, event, context);
+    }
+
+    fn key_up(&mut self, event: KeyEvent, context: &mut dyn AnyRasterContext) {
+        T::key_up(self, event, context);
+    }
+
+    // TODO key_pressed
```

Comment:
What is `key_pressed`? Are we sure we're going to want that instead of `character_received`?

Reply:
`character_recieved` was removed from winit, but yeah, not sure if we actually need a separate function from `key_down`, even web has deprecated it.